### ISSUE TITLE
Polygons aren't drawn the same

### ISSUE THREAD

**Issue by vinh-00** [python-pillow/Pillow#6908]
Hi,
I'm currently trying to migrate py2 to py3, from Pillow 6.2.1 to 9.3.0
I'm noticing some strange behaviors in the draw polygon functions, where my inputs are the same, but the outputs are slightly different. 
Heres a test case below, to run on py2.7 remove the width arg.
 
```python
from PIL import Image, ImageDraw
import numpy as np

def pil_test():

  poly = [(402,  52),(402,  52),(406,  54),(405,  55)]

  img = Image.new('I', (500, 500))
  draw = ImageDraw.Draw(img, mode='I')
  draw.polygon(poly, fill=100, width=0) 
  ## py2 - draw.polygon(poly, fill=100)
  
  iarr = np.array(img)
  print(iarr[52,403])
```
Pillow 6.2.1 output >> 0
Pillow 9.3.0 output >> 100

I would like to know if theres a workaround for this? or why is this happening?

**Comment by Yay295** [comment]
The default `width` for `polygon()` is 1. Have you tried using the default?

**Comment by radarhere** [comment]
Testing, I find that the difference is because of #5980. This resolved #4347, where there was a request that the following polygon not be discontiguous.

<img width="332" alt="Screen Shot 2020-01-08 at 4 58 26 PM" src="https://user-images.githubusercontent.com/2703145/72019827-66a70280-3238-11ea-8f1b-8a209452e873.png">

So, if your question is just a general one about why the latest version of Pillow might draw a polygon differently to an earlier version of Pillow, that is why. The change was introduced in Pillow 9.1.0, so if you would like to upgrade to Python 3 without including this change in Pillow, you can use Pillow 9.0.0 instead.

**Comment by vinh-00** [comment]
Thank you for the explanation and workaround.
